### ISSUE TITLE
[Bug] Change Accessibility focus in Android

### ISSUE THREAD

**Issue by Poli97** [xamarin/xamarin-android#4322]
### Description
Not able to change the accessibility focus on a certain element after a certain event (like a button press).
Asked also on [Community](https://forums.xamarin.com/discussion/178244/android-change-accessibility-focus)

### Steps to Reproduce

1. Create an Android project
2. Insert some random elements (like 3 buttons)
3. Handle the Click of one of the buttons
4. In the Click handler try to change the accessibility focus on one of the other buttons
5. Test the app on a physical device with TalkBack ON 

### Expected Behavior
When you tap a certain button the screen reader should change the focus to the wanted button

### Actual Behavior
Using the expected command, `myview.SendAccessibilityEvent(EventTypes.ViewAccessibilityFocused)` the screen reader only read the accessibility name of `myview` without actually setting the focus on it, that remain on the  pressed button.
Using some other commands, like `myview.SendAccessibilityEvent(EventTypes.ViewFocused)` or `b3.SendAccessibilityEvent(EventTypes.WindowsChanged)` nothing happen.
Using some other command like `myview.SendAccessibilityEventUnchecked(AccessibilityEvent.WindowsChangeAccessibilityFocused)` (that should actually be the right one that work in Android Studio), it give the error `Argument 1: cannot convert from 'Android.Views.Accessibility.WindowsChange' to 'Android.Views.Accessibility.AccessibilityEvent`

### Other information
In Android Studio, using Java, to set the accessibility focus on a certain `myview` you have to do `myview.sendAccessibilityEvent(AccessibilityEvent.WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED)` and it works fine. In Xamarin, using C#, everything that I tried doesn't work properly.

### Reproduction Link

[Git repository example](https://github.com/Poli97/AndroidChangeFocus)

**Comment by jonpryor** [comment]
`EventTypes.ViewAccessibilityFocused` corresponds to [`AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED`](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent#TYPE_VIEW_ACCESSIBILITY_FOCUSED).

If you want to do an equivalent of `sendAccessibilityEvent(AccessibilityEvent.WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED)`, you will need to do:

```csharp
myview.SendAccessibilityEvent((EventTypes) (int) WindowsChange.AccessibilityFocused);
```

The perils of enumification...

**Comment by Poli97** [comment]
> `EventTypes.ViewAccessibilityFocused` corresponds to [`AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED`](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent#TYPE_VIEW_ACCESSIBILITY_FOCUSED).
> 
> If you want to do an equivalent of `sendAccessibilityEvent(AccessibilityEvent.WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED)`, you will need to do:
> 
> ```cs
> myview.SendAccessibilityEvent((EventTypes) (int) WindowsChange.AccessibilityFocused);
> ```
> 
> The perils of enumification...

Damn, I feel so stupid for haven't seen that. Thanks a lot!

**Comment by gugavaro** [comment]
I believe you can actually use:
myview.SendAccessibilityEvent(EventTypes.ViewAccessibilityFocused)

**Comment by Poli97** [comment]
> I believe you can actually use:
> myview.SendAccessibilityEvent(EventTypes.ViewAccessibilityFocused)

No, if you do that, as I said in "Actual Behavior", the screen reader just reads the name of myview, without actually setting the focus on it.

**Comment by gugavaro** [comment]
I believe you could do this:

myview.SendAccessibilityEvent(EventTypes.ViewHoverEnter)

**Comment by Poli97** [comment]
> I believe you could do this:
> 
> myview.SendAccessibilityEvent(EventTypes.ViewHoverEnter)

Oh, yes, that worked too. Thanks!

**Comment by brendanzagaeski** [comment]
In case it's helpful additional info, I believe the `AccessibilityEvent.WINDOWS_CHANGE_*` constant values are not intended to be used when dispatching accessibility events. I believe those values are instead only meant to be used to compare to the return value from [`AccessibilityEvent.getWindowChanges()`](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent#getWindowChanges()).

So I think it would be recommended to use one of the `AccessibilityEvent.TYPE_*` constant values when providing an argument to `SendAccessibilityEvent()`.

In this case, if you've previously found that `WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED` gives you the desired behavior, using `EventTypes.ViewHoverEnter` would be a good choice because `TYPE_VIEW_HOVER_ENTER` and `WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED` have the same integer value of 128 (0x00000080).

(Note also that `getWindowChanges()` and the `WINDOWS_CHANGE_*` constants are only available starting in Android API level 28, while [`sendAccessibilityEvent()`](https://developer.android.com/reference/android/view/View#sendAccessibilityEvent(int)) and the `TYPE_*` constants are available in older Android API levels.)

**Comment by Poli97** [comment]
> In case it's helpful additional info, I believe the `AccessibilityEvent.WINDOWS_CHANGE_*` constant values are not intended to be used when dispatching accessibility events. I believe those values are instead only meant to be used to compare to the return value from [`AccessibilityEvent.getWindowChanges()`](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent#getWindowChanges()).
> 
> So I think it would be recommended to use one of the `AccessibilityEvent.TYPE_*` constant values when providing an argument to `SendAccessibilityEvent()`.
> 
> In this case, if you've previously found that `WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED` gives you the desired behavior, using `EventTypes.ViewHoverEnter` would be a good choice because `TYPE_VIEW_HOVER_ENTER` and `WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED` have the same integer value of 128 (0x00000080).
> 
> (Note also that `getWindowChanges()` and the `WINDOWS_CHANGE_*` constants are only available starting in Android API level 28, while [`sendAccessibilityEvent()`](https://developer.android.com/reference/android/view/View#sendAccessibilityEvent(int)) and the `TYPE_*` constants are available in older Android API levels.)

Oh I see. Perfect, very clear and exahustive, really appreciate that thanks.
